Create profile card

alissatroiano opened the pull request:
# Hackathon Git Labs Project PR 🎉🎉

## PR contains:
- Updated `Alissa` directory with proper spelling so links work
- Updated `Alissa/index.html` with profile card html
- Updated `Alissa/style.css` with profile card styles
- New button on the bottom of the page that links back to **main** `index.html`
-  ***Note: This page is still in progress, and there is commented out code in these files that will be removed when page is completed***

## Breaking changes
- None

## Screenshots
<img width="686" alt="Screen Shot 2022-03-07 at 9 33 56 AM" src="https://user-images.githubusercontent.com/56362524/157054549-3cb26a19-34ae-4ebc-a56c-838cae0d6b18.png">

auxfuse reviewed (changes requested):
Hi Alissa! Great work here 😄 My only concern is the .DS_Store files? Can they be removed please? 😀 Once done, I'll merge the PR straight away. I've also updated the gitignore file now too to suit.

alissatroiano commented:
> Hi Alissa! Great work here 😄 My only concern is the .DS_Store files? Can they be removed please? 😀 Once done, I'll merge the PR straight away. I've also updated the gitignore file now too to suit.

@auxfuse 

I thought this PR was closed because I didn't reload the browser, so I actually deleted this branch. I added those annoying MACOS DS_Store files to the .gitignore on the other branch I made (in the next PR). So we can close this one and merge that other one
